docs/reference/reference-architectures/hot-frozen.asciidoc

```diff
@@ -0,0 +1,152 @@
+[[hot-frozen-architecture]]
+== Hot / Frozen - High Availability
+
+The Hot / Frozen high availability architecture is cost optimized for large time-series datasets. In this architecture, the hot tier is primarily used for indexing, searching, and continuity for automated processes. https://www.elastic.co/guide/en/elasticsearch/reference/current/searchable-snapshots.html[Searchable snapshots] are taken from hot into a repository, such as a cloud Object Store or an on-premesis shared filesystem, and then cached to any desired volume on the local disks of the Frozen tier. Data in the repository is indexed for fast retrieval and accessed on-demand from the Frozen nodes. Index and Snapshot lifecycle management are used to automate this process. 
```

Comment:
```suggestion
The Hot / Frozen High Availability architecture is cost optimized for large time-series datasets. In this architecture, the hot tier is primarily used for indexing, searching, and continuity for automated processes. https://www.elastic.co/guide/en/elasticsearch/reference/current/searchable-snapshots.html[Searchable snapshots] are taken from hot into a repository, such as a cloud object store or an on-premise shared filesystem, and then cached to any desired volume on the local disks of the frozen tier. Data in the repository is indexed for fast retrieval and accessed on-demand from the frozen nodes. Index and snapshot lifecycle management are used to automate this process. 
```

I checked the Elasticsearch docs and it seems we don't capitalize the [data tier types](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-tiers.html) nor [ILM/SLM](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-lifecycle-management.html).